applications/validate_optics.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
+
+import logging
+import matplotlib.pyplot as plt
+import argparse
+import numpy as np
+
+import astropy.units as u
+
+import simtools.config as cfg
+import simtools.util.general as gen
+from simtools.model.telescope_model import TelescopeModel
+from simtools.ray_tracing import RayTracing
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(
+        description=(
```

Comment:
Is this really the description or is it left over from previous code?

Reply:
Left over, sorry.